src/generators/dom/visitors/YieldTag.js

```diff
@@ -1,9 +1,11 @@
 export default function visitYieldTag ( generator, block, state ) {
-	block.builders.mount.addLine(
-		`${block.component}._yield && ${block.component}._yield.mount( ${state.parentNode || block.target}, null );`
+	const parentNode = state.parentNode || block.target;
+
+	( state.parentNode ? block.builders.create : block.builders.mount ).addLine(
+		`if ( ${block.component}._yield ) ${block.component}._yield.mount( ${parentNode}, null );`
```

Comment:
No particular reason, just saw it and thought there's no reason it *shouldn't* be an `if` — the intent is clearer, and it minifies the same way. I reckon we should generally prefer `if` to `&&` for statements (as opposed to expressions). Don't feel all that strongly though